README.md

```diff
@@ -1,12 +1,35 @@
-# Node SDK for Crowdin API v2
+[<p align="center"><img src="https://support.crowdin.com/assets/logos/crowdin-dark-symbol.png" data-canonical-src="https://support.crowdin.com/assets/logos/crowdin-dark-symbol.png" width="200" height="200" align="center"/></p>](https://crowdin.com)
 
-Node.js client for integration with Crowdin API v2.
+# Crowdin JavaScript client
 
-For more about Crowdin API v2 see [Crowdin API](https://support.crowdin.com/enterprise/api/)
+The Crowdin JavaScript client is a lightweight interface to the Crowdin API v2 that works in both Node.js and browser environments. It provides common services for making API requests.
```

Comment:
Need to correct statement with environments. Actually in ANY environment because (for example) Sketch plugins are running not either in Node or browser env.